MaxText/layers/linears.py

```diff
@@ -385,27 +385,25 @@ def gmm(inputs, kernel, group_sizes):
         shard_map.shard_map,
         mesh=self.mesh,
         in_specs=(
-              (nn.logical_to_mesh_axes((BATCH, None, EMBED))),
               (nn.logical_to_mesh_axes((BATCH, None, None))),
```

Comment:
Can we add a comment here (or at top of annotations) saying we purposefully AG the input activations over TP to follow a specific sharding strategy outlined in https://parsa.epfl.ch/course-info/cs723/papers/Megatron.pdf 

(we can include this link the comment)

Reply:
Sure thing, added